bitnami/spring-cloud-dataflow/README.md

```diff
@@ -750,8 +750,8 @@ $ helm upgrade dataflow bitnami/spring-cloud-dataflow \
 
 You should see the lines below in MariaDB container logs:
 
-```console
-$ kubectl logs $(kubectl get pods -l app.kubernetes.io/instance=dataflow,app.kubernetes.io/name=mariadb,app.kubernetes.io/component=primary -o jsonpath="{.items[0].metadata.name}")
+```bash
+kubectl logs $(kubectl get pods -l app.kubernetes.io/instance=dataflow,app.kubernetes.io/name=mariadb,app.kubernetes.io/component=primary -o jsonpath="{.items[0].metadata.name}")
```

Comment:
I would revert the changes in this particular case. The rest of code blocks are just telling the user the command to use, however this one is showing an example including the command's output. For this particular case, using `console` + `$` is the right choice since the formatting is more appropriate.

Reply:
Agreed. Adjusted.